igvm/hypervisor.py

```diff
@@ -624,55 +641,13 @@ def vm_info(self, vm):
         props = DomainProperties.from_running(self, vm, self._get_domain(vm))
         return props.info()
 
-    def get_logical_volumes(self):
-        lvolumes = []
-        lvs = self.run(
-            'lvs --noheadings -o name,vg_name,lv_size --unit b --nosuffix'
-            ' 2>/dev/null',
-            silent=True
-        )
-        for lv_line in lvs.splitlines():
-            lv_name, vg_name, lv_size = lv_line.split()
-            lvolumes.append({
-                'path': '/dev/{}/{}'.format(vg_name, lv_name),
-                'name': lv_name,
-                'vg_name': vg_name,
-                'size_MiB': math.ceil(float(lv_size) / 1024 ** 2),
-            })
-        return lvolumes
-
-    def lv_create(self, lv_name, size_gib):
-        self.run('lvcreate -y -L {}g -n {} {}'.format(
-            size_gib,
-            lv_name,
-            VG_NAME,
-        ))
-
-    def lv_remove(self, lv_path):
-        self.run('lvremove -f {}'.format(lv_path))
-
-    def lv_resize(self, lv_path, size_gib):
-        """Extend the volume, return the new size"""
-        self.run('lvresize {} -L {}g'.format(lv_path, size_gib))
-
-    def lv_rename(self, lv_name, new_lv_name):
-        if lv_name != new_lv_name:
-            self.run('lvrename {} {} {}'.format(VG_NAME, lv_name, new_lv_name))
-
```

Comment:
:partyparrot:

Reply:
Just simplified by joining two functions into one.

Reply:
He's expired and gone to meet its maker!